platform/utils/endpoints-generator/endpoints.js

```diff
@@ -1,7 +1,19 @@
+const fallbackURL = "https://localhost:8000";
+const fallbackAuth = "";
+
 export default {
-    polyPediaReport: {
-        url: process.env.POLYPOD_POLYPEDIA_REPORT_URL,
-        auth: process.env.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION || "",
+    "polyPediaReport/facebookImporter": {
+        url:
+            process.env.POLYPOD_POLYPEDIA_REPORT_URL ||
```

Comment:
So this change is just meant to add another entry for google in `endpoints.json`?
Because as I see, google and facebook, will still use the same `POLYPOD_POLYPEDIA_REPORT_URL`, right?

Reply:
No. we will not actually. Talked about it with Tom and he adds `<report_url>/facebookImport` and `<report_url>/google`. Makes it easier